pxd_io_uring.h

```diff
@@ -63,6 +63,7 @@ struct io_uring_sqe {
 #define IORING_OP_COPY_DATA	10
 #define IORING_OP_DISCARD_FIXED 11
 #define IORING_OP_SYNCFS_FIXED  12
+#define IORING_OP_DATA_SWITCH  13
```

Comment:
DATA_SWITCH is a bit hard to guess what it means. Maybe READ_BIO/WRITE_BIO or READ_REQUEST/WRITE_REQUEST to continue with the theme READV/READ_FIXED.

Reply:
Since we are dealing with the external BIO directly, direction is already known from the associated bio's bio_data_dir(). And providing READ/WRITE options may allow to inadvertently perform opposite operation on end user bio, thereby corrupting data buffers. That is why a READ/WRITE version was not provided.

But I can rename this to RWBIO, lets note, that no data direction is supplied from user space, it is implicitly deduced from the end user bio waiting in kernel.